Register - Signing off trainee teachers for the census (#1642)
DFE-Digital/bat-design-history: app/posts/register-trainee-teachers/2025-06-13-census-sign-off.md

---
title: Census sign-off
description: Signing off trainee teachers for the census
date: 2025-06-13
tags:
  - trainees
  - providers
  - census
---

> At the beginning of each academic year, between 1st September and 31st October, Initial Teacher Training (ITT) data must be submitted to Register trainee teachers (Register) and signed off, as the Department for Education (DfE) requires accurate data for the ITT census publication.

Reviewer: ```suggestion
At the beginning of each academic year, between 1st September and 31st October, initial teacher training (ITT) data must be submitted to Register trainee teachers (Register) and signed off, as the Department for Education (DfE) requires accurate data for the ITT census publication.
```

## How it works

The initial step will remain unchanged. Each year, the DfE will continue to notify providers by email that the census sign-off window is open and that they should log in to Register to confirm their new trainee numbers and complete the sign-off process.

Once logged into Register, a notification banner will appear on the home screen during the census sign-off period if no one in the organisation has signed off on the census process. This banner will inform providers that the census sign-off is due and provide a link to the reports page.

![Screenshot showing an example banner on the home page notifying that the census sign off is due](example-home-banner.png "Census sign-off notification banner example")

From the reports page, providers can access the census sign-off process. If someone from the organisation has already signed off the census, they will see the following message indicating that the sign-off has been completed.

> Someone in your organisation has already completed the census sign-off.

![Screenshot showing the reports page](reports.png "Reports page")

> The next page offers guidance on completing the census sign-off process, including a link to further information on the [Initial Teacher Training Census methodology](https://explore-education-statistics.service.gov.uk/methodology/initial-teacher-training-census). Users can then download a CSV file of the new trainees that Register currently holds for the organisation for the academic year, allowing them to check against their student records system (SRS) to ensure the numbers are accurate.

Reviewer: ```suggestion
The next page offers guidance on completing the census sign-off process, including a link to further information on the [initial teacher training census methodology](https://explore-education-statistics.service.gov.uk/methodology/initial-teacher-training-census). Users can then download a CSV file of the new trainees that Register currently holds for the organisation for the academic year, allowing them to check against their student records system (SRS) to ensure the numbers are accurate.
```

![Screenshot showing an alternative navigation](guidance.png "Guidance page")

The following page is the actual sign-off page, where users can confirm that the numbers are accurate.

The census sign off summary card includes:

- provider name

> - UK Provider Reference Number (UKPRN)

Reviewer: ```suggestion
- UK provider reference number (UKPRN)
```

- currently logged-in approver's name

The checkbox must be checked in order to confirm that the trainee numbers are correct.

> Once the 'Sign off census' button is pressed to complete the process, an email will be sent to all users for that provider, confirming that the process has been completed on their behalf.

Reviewer: ```suggestion
Once the ‘Sign off census’ button is pressed to complete the process, an email will be sent to all users for that provider, confirming that the process has been completed on their behalf.
```
